cmd/operator-sdk/add/api.go

```diff
@@ -25,7 +25,6 @@ import (
 	"github.com/operator-framework/operator-sdk/internal/scaffold/input"
```

Comment:
Any `fmt.Errorf` in `cmd/operator-sdk` can be left as `%v` for now since the consumer of those errors is cobra, which doesn't unwrap errors. You can remove the parentheses around `(%v)` though.

Reply:
I can do a find-replace all for %w with %v for all files under `cmd/operator-sdk`, but was confused with another [comment below](https://github.com/operator-framework/operator-sdk/pull/2355/files#r362047287)

Reply:
Done